### ISSUE TITLE
bug(zod-x/prefer-namespace-import): v3 & v4 imports

### ISSUE THREAD

**Issue by derekcicerone** [marcalexiei/eslint-plugin-zod-x#93]
### 🧩 Description

This is a great project!

We have a large codebase that uses 3rd party functionality which still requires using zod v3 so we have code like this:
```
import * as z3 from "zod/v3";
import * as z from "zod/v4";
```

That code shows this error:
<img width="617" height="83" alt="Image" src="https://github.com/user-attachments/assets/f2c6cff8-f0bb-4e1e-8fb6-59febed85249" />

### 🪜 How to reproduce

See above.

### ✅ Expected behavior

I believe the sample code above should work with the `zod-x/prefer-namespace-import` rule enabled.

### 💻 System info

```shell
all standard stuff
```

### ✒️ Additional context

Happy to provide more details if needed.

**Comment by marcalexiei** [comment]
Hello Derek,

> This is a great project!

🙏 Thank you very much 🙏

---

To solve the issue I'm gonna treat each import source separately with the same logic in place right now:

```ts
import z3 from "zod/v3";
import z from "zod/v4";
```

⬇️ FIX ⬇️

```ts
import * as z3 from "zod/v3";
import * as z from "zod/v4";
```

---

```ts
import * as z3 from 'zod/v3';
import { string } from 'zod/v3'; // Error: Zod is already imported via namespace import
import * as z from 'zod/v4';
```

⬇️ FIX ⬇️

```ts
import * as z3 from "zod/v3";
import * as z from "zod/v4";
```


I'll work on a PR in the upcoming days

**Comment by marcalexiei** [comment]
Fix available in [1.8.1](https://github.com/marcalexiei/eslint-plugin-zod-x/releases/tag/v1.8.1).

Thanks for the report!
